client/src/components/ModalView.jsx

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React from 'react';
```

Comment:
I believe you can fix these linting errors by doing `import React, { useState } from 'react'`
Or you can try running `yarn lint:fix`, or follow Jeffrey's [guide](https://github.com/hack4impact-uiuc/falling-fruit/wiki/VS-Code-Setup-(for-auto-linting-and-formatting)) to have your VSCode format on save